doc/pylint.rst

```diff
@@ -75,9 +80,11 @@ Advanced options
 You can turn certain messages off at the line, block or file/module level by adding a ``# pylint: disable=MESSAGE-NAMES`` comment at the respective `scope`_, where ``MESSAGE_NAMES`` should be replaced with a comma-separated list (or single value) of `Pylint message names`_.
 For example, a directive might look like ``# pylint: disable=multiple-statements``, or ``# pylint: disable=fixme, line-too-long``. 
 
+.. _scope: http://pylint.pycqa.org/en/latest/user_guide/message-control.html
+
```

Comment:
```suggestion
```

This is already linked below. My suggested course of action is to remove the inline link here, not add another one.

Reply:
But maybe its better to link it here and link something more general for the Pylint docs? Not sure, we can discuss.

Reply:
Rejected/superseded.